app/assets/javascripts/components/LanguageSearchBar.es6.jsx

```diff
@@ -0,0 +1,49 @@
+class LanguageSearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getItemTitle(item) {
+    // default to glottocode if there are no identifiers
+    return item.matched_identifiers.length == 0 ? item.glottocode : item.matched_identifiers[0];
+  }
+
+  renderLanguageSuggestion(item, isHighlighted) {
+    // TODO couldn't figure out how to call getItemTitle here, even after binding it in constructor
+    const title = item.matched_identifiers.length == 0 ? item.glottocode : item.matched_identifiers[0];
```

Comment:
This line is the same as in the `getItemTitle` function, which I couldn't properly call for some reason.